csrc/scheduler/multi_matmul.cpp

```diff
@@ -1006,6 +1006,37 @@ class MultipleMatmulScheduler {
     scheduleBranch(bs_, bcw_smems_, params_->supported_vec_size.b);
   }
 
+  // Schedule acr and bcr
+  void scheduleLdMatrixLoads() {
```

Comment:
Hmm... Looks like this and `scheduleWarpTile` is still not used?

Reply:
You're right again. Removing it again. Sorry about that.